policyengine_us/parameters/gov/states/ny/tax/income/credits/ctc/minimum_age.yaml

```diff
@@ -1,6 +1,7 @@
 description: New York's Empire State Child Credit minimum age qualification.
 values:
   2018-01-01: 4
+  2023-01-01: 0
```

Comment:
Could you check that the code is >= and update the description like other parameters?